carbonmark-api/test/routes/retirements/[id]/provenance/get.test.mocks.ts

```diff
@@ -0,0 +1,11 @@
+import nock from "nock";
+import { GRAPH_URLS } from "../../../../../src/app.constants";
+import { fixtures } from "../../../../fixtures";
+const mockProvenanceRecord = fixtures.digitalCarbon.provenanceRecord;
+
+export const mockDigitalCarbonProvenanceRecords = () =>
+  nock(GRAPH_URLS["polygon"].digitalCarbon)
+    .post("")
```

Comment:
Please add the query name lookup so that we dont trample on other requests to `.digitalCarbon`

As an example:

https://github.com/KlimaDAO/klimadao/blob/512ee69aa794e276bcd9cf0526360ff99cb11960/carbonmark-api/test/routes/projects/get.test.mocks.ts#L61

Reply:
Done in cea91c911